### ISSUE TITLE
Quizzes closing immediately

### ISSUE THREAD

**Issue by glherman** [PrairieLearn/PrairieLearn#3226]
Two students are reporting that their quizzes are closing immediately after completing the honor code. I can verify that this seems to be happening.

See netid erzonca2 and jackvn2 for Quiz 4 and 5 of CS 233 FA20 for examples

> Hi! I'm really sorry this has happened for the second time now, but when I went to take quiz 5, just like when I first tried quiz 4, it had me check the box to confirm I wasn't receiving outside help, and then immediately said the quiz was closed. I haven't taken the quiz yet, could it please be reopened?

> I think the issue is with my browser - I use Firefox with a lot of different security settings, and one of them stops it from syncing to my local time. It just uses GMT, so it assumes I'm several hours ahead of where I actually am. I think this is causing PrairieLearn to think I've run out of time for the quiz. I will work on figuring out which of my settings is changing the time, but in the meantime I'll start taking quizzes on Chrome and see if that helps.

> Thank you!

**Comment by echuber2** [comment]
Maybe related to #3227

**Comment by glherman** [comment]
yep, definitely seems to be a firefox issue as another student just reported the same issue and was on Firefox

**Comment by cheeren** [comment]
This also happened for one of our students, but seemed to be browser

**Comment by reteps** [comment]
Closing this out as I haven't heard anything similar to this recently.
